src/Model/FilterJob.re

```diff
@@ -135,35 +136,28 @@ module Make = (Config: Config) => {
     (false, newPendingWork, completed);
   };
 
+  /* Compare two ranked items. */
+  let compareItems = (a, b) => compare(a.Filter.score, b.Filter.score);
+
   let doActualWork =
       (
         {queue, shouldLower, filter, explodedFilter, _} as pendingWork: PendingWork.t,
-        {filtered, _}: CompletedWork.t,
+        {ranked, _}: CompletedWork.t,
       ) => {
     // Take out the items to process this frame
     let (items, queue) = Queue.take(Constants.itemsPerFrame, queue);
 
-    // Do a first filter pass to check if the item satisifies the regex
-    let filtered =
-      List.fold_left(
-        (acc, item) => {
-          let name = format(item, ~shouldLower);
-
-          if (Filter.fuzzyMatches(explodedFilter, name)) {
-            [item, ...acc];
-          } else {
-            acc;
-          };
-        },
-        filtered,
-        items,
-      );
```

Comment:
I'm not convinced we need this now, with how fast `fzy` is.

Reply:
Nice! It'll be great to simplify this